integrations/sonarqube/integration.py

```diff
@@ -184,6 +184,11 @@ def default_metrics() -> list[str]:
         metrics: list[str] = Field(
             description="List of metric keys", default=default_metrics()
         )
+        use_internal_api: bool = Field(
+            alias="useInternalApi",
+            description="Use internal API to fetch more data",
+            default=False,
```

Comment:
Should be true, to be backwards compatible 

And the default template mapping and blueprints should use the False option

Reply:
@lordsarcastic